README.md

```diff
@@ -12,6 +12,7 @@ This repository holds several references to example workflows and demonstrates h
 | ------------------------------------------------------------ | ------------------------- | ---------------- |
 |[cloudrun-docker](workflows/deploy-cloudrun/cloudrun-docker.yml) | ✅ | Build a Docker container, publish it to Google Artifact Registry, and deploy to Google Cloud Run. |
 |[cloudrun-source](workflows/deploy-cloudrun/cloudrun-source.yml) | ✅ | Deploy to Google Cloud Run directly from source. |
+|[cloudrun-declarative](workflows/deploy-cloudrun/cloudrun-declarative.yml) | ✅ | Build a Docker container, publish it to Google Artifact Registry, and deploy to Google Cloud Run using a [declarative YAML Service specification (KRM)](https://cloud.google.com/sdk/gcloud/reference/run/services/replace).  |
```

Comment:
Did you manually add this? Can you delete this line and run `go run scripts/generate.go readme`?

Reply:
yes I created that line manually - resolved by running the script